src/gui/windows/debug.zig

```diff
@@ -48,11 +50,17 @@ fn flawedRender() !void {
 		y += 8;
 		// TODO: packet loss
 		// TODO: Protocol statistics(maybe?)
+                try draw.print("Connections number: {}", .{main.game.world.?.manager.connections.items.len},  0, y, 8, .left);
+                y += 8;
+                try draw.print("PacketsSent number: {}", .{Connection.packetsSent.value},  0, y, 8, .left);
```

Comment:
`Connection.packetsSent.value`
That's not the right way to load an atomic value.
Correct would be `Connection.packetsSent.value.load()` with the correct memory order. I think I used `Monotonic` for these, but better double check it.